Assets/MixedRealityToolkit/MixedReality.Toolkit.targets

```diff
@@ -1,14 +1,32 @@
 <Project>
 
-  <PropertyGroup>
+  <PropertyGroup Condition="'$(MSBuildForUnityVersion)' == ''">
```

Comment:
Please add a comment as to why this Condition is here, as in "Check if MSBuildForUnity is being used to resolve, by checking if it's version is present or not"